06_iito/index.tex

```diff
@@ -1 +1,34 @@
+
+\textbf{Common Pool Donations: }Allowing islands to broadcast how much they plan to donate to the common pool lets them declare to the other islands if they plan to donate more than the specified tax amount put forward by the president as a form of virtue signalling. This may help an island redeem itself if it had previously lost trust. An island can lie about the amount it will donate if it is trying to maliciously gain favour but this can be caught by the Judge if they choose to inspect it, which may result in punishment.
+
+\textbf{Voting History: }Islands can request voting history from other islands or provide their own unprompted. This lets islands try to verify that the Speaker has been honest when counting votes but the islands can lie in the history that they provide, meaning islands may want to only take heed of information from islands they already trust. 
+
+\textbf{Tax History: }In a similar sense, islands can request taxation history (how much the islands were told to give in tax by the President) from other islands but an island may choose to not provide this history or be dishonest about it. If the islands are honest, this allows for more transparency and lets the islands better judge if the President is performing their role correctly.
+
+\textbf{Current Resources: }Islands are also able to share the value of their current resources. This may assist islands when making decisions regarding gifts (Section~\ref{sec:IITO:gifting}). For example, if an island requests a gift because they are low on resources, the island receiving the request may want to ask how many resources the requesting island has. This can also give an island an indication of the overall success of the archipelago. Similarly to intended common pool donations and voting and tax history sharing, islands can lie or withhold this information.
+
+\section{Gifting}  
+\label{sec:IITO:gifting}  
+
+If an island is struggling and requires resources to survive it may ask the President for an allocation from the common pool but the President may reject this request. The struggling island would still be able to take money from the common pool if rejected but law-abiding islands would probably want to avoid this. To give them another option they can request resources in the form of a gift from the other islands. Islands may accept this request if they view the survival of the requesting islands as beneficial or if they wish to improve their standing with the requesting islands. When requesting, giving or receiving a gift, the island can specify a reason for their action, which gives islands more information about the transaction. For example, a requesting island may want to specify that they will move to a critical state in the next round if they do not receive the gift or an offering island may want to specify that the gift is meant as a reward for successful disaster forecasting.
```

Comment:
suggested re-writing:

If an island is struggling and requires some additional amount of resources to survive, it may ask the President for an allocation from the common pool but the President may reject this request. The struggling island would still be able to take resources from the common pool if rejected, but law-abiding islands would probably want to avoid such disobedience. To allow islands to still request an additional amount of resources from the other islands, \textbf{gifting} action is an option. Islands may accept the gift requests if they wish to improve their standing with the requesting islands. They can also accept the gift requests of those whose survival is deemed to be essential for the future of the archipelago. When requesting, giving or receiving a gift, the island can specify a reason for this action, which gives islands more information about the gifting transaction. For example, a requesting island may want to specify that they will move to a critical state in the next round if they do not receive the gift, or an offering island may want to specify that the gift is meant as a reward for successful disaster forecasting.